core/src/test/java/org/elasticsearch/ingest/IngestClientIT.java

```diff
@@ -60,28 +59,6 @@
         return pluginList(IngestPlugin.class);
```

Comment:
That does sound better. It will require some work (moving that test case out from the client smoke qa test, maybe adding a client-test gradle plugin like we have rest-test). Can be a followup.